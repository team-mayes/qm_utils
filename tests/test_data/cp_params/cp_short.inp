14b.sdf  0.767  -1.253   0.241  1.279   0.000  -0.482  0.767   1.253   0.241 -0.767   1.253   0.241 -1.279   0.000  -0.482 -0.767  -1.253   0.241
3e.sdf  0.687  -1.183  -0.121  1.391   0.021  -0.002  0.632   1.175   0.227 -0.651   1.119  -0.329 -1.352   0.044   0.231 -0.708  -1.176  -0.005
