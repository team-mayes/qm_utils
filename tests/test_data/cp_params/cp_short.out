"name","phi","theta","q","closest_pucker"
"14b.sdf",240.00000000000003,90.0,0.8348484892481989,"14b"
"3e.sdf",0.5165961211938987,125.16673486810083,0.4766146774356231,"3e"
